MapboxDirections/MBRouteOptions.swift

```diff
@@ -304,6 +309,13 @@ open class RouteOptions: NSObject, NSSecureCoding {
     open var includesExitRoundaboutManeuver = false
     
     /**
+     An Array of `RoadClass` that the routing engine will attempt to avoid.
+     
+     The order of this Array does not impact the results.
+     */
+    open var excludedRoadClasses: RoadClasses = []
```

Comment:
Sorry for the indecision here, but after writing the documentation suggestion above, it just occurred to me that `roadClassesToAvoid` would be even more readily understood than `excludedRoadClasses`.